src/dbnode/client/fetch_tagged_results_accumulator.go

```diff
@@ -211,8 +211,8 @@ func (accum *fetchTaggedResultAccumulator) accumulatedResult(
 			accum.errors)
 		err := xerrors.Wrapf(
 			consistencyErr,
-			"unable to satisfy consistency requirements: shards=%d, err=%v",
-			accum.numShardsPending, consistencyErr)
+			"unable to satisfy consistency requirements, shards=%d",
+			accum.numShardsPending)
```

Comment:
`Wrap()` and `Wrapf()` automatically append the inner error message, don't need to do it manually

https://github.com/m3db/m3/blob/f4e32f95bb6e0b9f69ed22a2eedef6057a2bcd27/src/x/errors/errors.go#L90-L101